pkg/scheduler/plugins/reservation/hook.go

```diff
@@ -159,7 +159,8 @@ func (h *Hook) prepareMatchReservationState(handle frameworkext.ExtendedHandle,
 				klog.V(5).Infof("unable to convert to *schedulingv1alpha1.Reservation, obj %T", obj)
 				continue
 			}
-			if !IsReservationScheduled(r) { // only count scheduled reservations
+			// only count scheduled & available reservations, ignore succeeded ones
+			if !IsReservationScheduled(r) {
```

Comment:
The name `IsReservationScheduled` is confusing,`Scheduled` and `Available` are two different states. `Available` requires that the Reservation MUST BE successfully scheduled and there are remaining resources to allocate, but if there are no resources to allocate, it is also successfully scheduled. Why not just call it `IsReservationAllocatable` or `IsReservationAvailable`?

Reply:
OK. It is renamed `IsReservationAvailable`.